modules/dnn/src/layers/elementwise_layers.cpp

```diff
@@ -577,12 +578,29 @@ struct ReLU6Functor : public BaseFunctor
     }
 #endif  // HAVE_DNN_NGRAPH
 
+
+
 #ifdef HAVE_WEBNN
+    ml::Operand BuildConstant(const ml::GraphBuilder& builder,
+                              const std::vector<int32_t>& dimensions,
+                              const void* value,
+                              size_t size,
+                              ml::OperandType type) {
```

Comment:
Please refer to https://github.com/webmachinelearning/webnn-native/blob/main/examples/SampleUtils.h#L43, handle other types.